tests/helpers_tests/test_condition.py

```diff
@@ -25,22 +25,16 @@
 doubleCondition = ['pi_value', 3.14, 'custom_attribute', 'exact']
 
 semver_equal_2_0_0_condition_list = [['Android', "2.0.0", 'custom_attribute', 'semver_eq']]
-semver_equal_2_condition_list = [['Android', "2", 'custom_attribute', 'semver_eq']]
-semver_equal_2_0_condition_list = [['Android', "2.0", 'custom_attribute', 'semver_eq']]
 semver_equal_2_0_1_beta_condition_list = [['Android', "2.0.1-beta", 'custom_attribute', 'semver_eq']]
```

Comment:
I see other variables starting with prefix semver_ here. Are they being used? If not, we should remove them.